_inc/client/writing/speed-up-site.jsx

```diff
@@ -30,52 +32,87 @@ const SpeedUpSite = moduleSettingsForm(
 			}
 		};
 
+		handleCdnChange = () => {
+			const currentPhoton = this.props.getOptionValue( 'photon' );
+			const currentCdn = this.props.getOptionValue( 'photon-cdn' );
+
+			// Tiled Galleries depends on Photon. Deactivate it when Photon is deactivated.
+			this.props.updateOptions( {
```

Comment:
I spotted an issue with that logic, when trying to toggle the option off with only one of the modules active. This will need to be updated.